frontend/src/components/Request.jsx

```diff
@@ -125,18 +132,18 @@ export default function Request() {
       // If post is successful, redirect to home page
       redirect();
     } catch (error) {
-      // console.error(error);
+      setDataOffline(true);
     }
   };
 
   return (
     <div className="flex flex-row flex-wrap justify-center items-center">
       {contextHolder}
       {/* Render the package card on the screen next to the form */}
-      {packageData !== null && (
+      {packageData !== null && !dataOffline ? (
         <Package data={packageData} showRequest={false} />
-      )}
-      {personalCareProducts.length > 0 && (
+      ) : null}
+      {personalCareProducts.length > 0 && !dataOffline ? (
```

Comment:
```suggestion
      {personalCareProducts.length > 0 && (
```

revert; this is okay to show nothing if there are no personal care products